omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -441,6 +471,9 @@
 ####
 default['private_chef']['oc_chef_authz']['http_init_count'] = 25
 default['private_chef']['oc_chef_authz']['http_max_count'] = 100
+# The queue max is only effective if authz_pooler_timeout (in the
```

Comment:
I believe it does.  That timeout configurable feeds this in the app config:

https://github.com/chef/chef-server/blob/ssd/swimsuits-and-sunblock/omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb#L182-L184

which is read here and used in the take_member/2 function:

https://github.com/chef/opscoderl_httpc/blob/master/src/oc_httpc.erl#L115
https://github.com/chef/opscoderl_httpc/blob/master/src/oc_httpc.erl#L135-L145

The rest of the pool config is read in here:

https://github.com/chef/chef-server/blob/ssd/swimsuits-and-sunblock/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_http.erl#L201-L203

which is passed to oc_httpc:create_pool here:

https://github.com/chef/chef-server/blob/ssd/swimsuits-and-sunblock/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_http.erl#L191-L194

which explicitly calls the pooler:new_pool function here:

https://github.com/chef/chef-server/blob/ssd/swimsuits-and-sunblock/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_http.erl#L191-L194

I too find it odd that we didn't just use the pooler config for this.